eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -10,8 +10,8 @@ param (
 
   # arguments necessary to power the docs release
   $DocRepoLocation, # the location on disk where we have cloned the documentation repository
-  $Language, # EG: js, java, dotnet. Used in language for the embedded readme.
-  $DocRepoContentLocation = "docs-ref-services/" # within the doc repo, where does our readme go?
+  $Language # EG: js, java, dotnet. Used in language for the embedded readme.
```

Comment:
Isn't this link missing a common after $Languge?

Reply:
Yessir! This will become clear after you see the PRs I submit in the next couple minutes here.